solvers/gist.py

```diff
@@ -0,0 +1,104 @@
+from benchopt import BaseSolver, safe_import_context
+
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from numpy.linalg import norm
+    from numba import njit
+
+if import_ctx.failed_import:
+
+    def njit(f):  # noqa: F811
+        return f
+
+
+@njit
```

Comment:
all operations are vectorized in this solver. I don't think jitting brings anything, and I am in favor in removing if from the whole solver.